Resources/Textures/Objects/Weapons/Guns/Rifles/lecter.rsi/meta.json

```
@@ -1,11 +1,11 @@
 {
-    "version": 1,
-    "license": "CC-BY-SA-3.0",
-    "copyright": "Taken/modified from tgstation at https://github.com/tgstation/tgstation/pull/41393/commits/1e56473177d0994d163c9edca3d13d6e5b640cc4, https://github.com/tgstation/tgstation/tree/master/icons/obj/weapons/guns backpack sprite by Peptide (copy of carbine), backpack sling sprite edited by Boaz1111, wield sprites by RiceMar1244",
-    "size": {
-        "x": 32,
-        "y": 32
-    },
+	"version": 1,
+	"license": "CC-BY-SA-3.0",
+	"copyright": "All sprites by https://github.com/august-sun.",
```

minor attribution nitpick but any way you go is fine as long as you give you github username and explicitly say it's github
```suggestion
	"copyright": "All sprites by august-sun (GitHub)",
```